### ISSUE TITLE
How do I open the samples/core/ project in Visual Studio 2019?

### ISSUE THREAD

**Issue by i1221** [dotnet/EntityFramework.Docs#3721]
How do I open the samples/core/ project in Visual Studio 2019?
https://github.com/dotnet/EntityFramework.Docs/blob/main/samples/core/Samples.sln

**Comment by roji** [comment]
@i1221 you would typically clone the `dotnet/EntityFramework.Docs` repo with git (you can use the "Code" button as in the screenshot below). Once that's done, you can directly open the Samples.sln in VS, or the csproj of a specific project which interests you.

![image](https://user-images.githubusercontent.com/1862641/152595422-9d98545c-fa9e-46f5-ab46-9e6fb070264c.png)

**Comment by i1221** [comment]
@roji 
I don't quite understand what needs to be done?
I need to lead the team somewhere"git@github.com:dotnet/EntityFramework.Docs.git "?
I'm using Windows 10.


I did as shown in my drawing.
I got the archive "EntityFramework.Docs-main.zip" 
 
I got
..\..\Entity Framework.Docs-main\samples\core\Samples.sln
I'm trying to open Samples.sln in VS 2019  
I get the message:
 _"The version number of the selected solution is greater than the version number of the solution supported by this release of Visual Studio"_

![05 02 2022_13-02-13](https://user-images.githubusercontent.com/52816062/152637964-ac0658a7-2a26-4d94-9443-b013ec36518a.png)
